src/utils/wrapInClass.js

```diff
@@ -1,17 +1,35 @@
-export default function wrapInClass(className, value) {
+import { stripIndents } from 'common-tags';
+
+export default function wrapInClass(text) {
   const imports = [];
+  const composes = [];
 
   let match;
-  const matcher = /@import.*?(?:$|;)/g;
+  const rImports = /@import.*?(?:$|;)/g;
+  const rComposes = /\b(?:composes\s*?:\s*(.*?)(?:from\s(.+?))?(?=[}\n\r]))/gim;
 
   // eslint-disable-next-line no-cond-assign
-  while ((match = matcher.exec(value))) {
+  while ((match = rImports.exec(text))) {
     imports.push(match[0]);
   }
+  match = null;
+  // eslint-disable-next-line no-cond-assign
+  while ((match = rComposes.exec(text))) {
+    composes.push(match[0]);
+  }
 
-  value = value.replace(matcher, '');
+  text = text.replace(rImports, '').replace(rComposes, '');
+
+  let val = stripIndents`
+    .cls1 {
+      ${text.trim()}
+    }`;
+
+  if (composes.length) {
+    composes.unshift(`composes: cls1;`);
```

Comment:
can you add a comment here explaining per your PR description why specifically `cls2` is required here? it makes sense, but it's not obvious at a glance